services/blockchain-indexer/shared/dataService/dpos/delegates.js

```diff
@@ -55,7 +55,7 @@ const DELEGATE_STATUS = {
 let delegateList = [];
 
 const delegateComparator = (a, b) => {
-	const diff = BigInt(b.delegateWeight) - BigInt(a.delegateWeight);
+	const diff = BigInt(b.voteWeight) - BigInt(a.voteWeight);
 	if (diff !== BigInt('0')) return Number(diff);
 	return a.address.localeCompare(b.address, 'en');
```

Comment:
Please ensure here that the delegate addresses here are not in Lisk32 format. We need them to be in hex format.